### PR TITLE
WIP:model: Add Vowpal wabbit

### LINKED ISSUE #328: model: vowpal wabbit
https://vowpalwabbit.org/tutorials/cmd_first_steps.html

### PR DESCRIPTION
**Pull request by 0dust**
Fixes #328

### CODE REVIEW

**Comment by 0dust** [comment]
@rahulunair Please have a look.

File: .ci/run.sh [hunk 1]
```diff
@@ -61,9 +55,6 @@ function run_plugin() {
     "${PYTHON}" -m unittest discover
     cd "${SRC_ROOT}"
 
-    # Deactivate venv
-    deactivate
```
Comment (pdxjohnny) [inline .ci/run.sh]: :+1:

**Comment by pdxjohnny** [comment]
Just FYI, that anaconda container looks like it runs Python 3.5 from the description. DFFML requires 3.7 minimum.

**Comment by 0dust** [comment]
> Just FYI, that anaconda container looks like it runs Python 3.5 from the description. DFFML requires 3.7 minimum.

Ah! Thanks.

**Pull request closed without merging**